GitUI/Translation/English.xlf

```diff
@@ -2315,7 +2315,7 @@ Do you want to remove it from the recent repositories list?</source>
         <target />
       </trans-unit>
       <trans-unit id="setNextPullActionAsDefaultToolStripMenuItem.Text">
-        <source>Set the next selection as default</source>
+        <source>Set default Pull button action</source>
```

Comment:
Given that text changed, please create a new translation key and remove the old one from all the files. Otherwise, on some languages UI will have invalid text.

Reply:
No need to change translations, they will get updated when we run `UpdateLocalEnglishTranslations.ps1` script
https://github.com/gitextensions/gitextensions/wiki/How-To:-prepare-a-release#2-preparing-to-release

Reply:
Hm, how does it work? We modified the text for the existing text, but it has the same key. So how would the translation tool know that it should abandon the translations from other files? Or you mean that we still need to apply the fix I asked for, but don't touch the other translation files?

Reply:
Changing the key seems like a forced workaround. Wouldn't it then be better to delete translations from all `*.xlf` files for this control? If that's necessary at all, I am not familiar with `UpdateLocalEnglishTranslations.ps1`

Reply:
Why do you think it's a workaround? You changed the meaning and purpose of the string completely. It's a brand new string and it's quite natural that is has a new identity.

Reply:
It's because the website translates the text and not the key. The script processes all translation strings and updates the English.xlf. That is uploaded to the website. The website updates xlf files for other languages.

Reply:
I invite you to read the whole page, it explains how we do major releases, and what the process entails.
In the nutshell, English.xlf gets regenerated from the code. Then, like Jay said, we upload it to Transifex, where it gets translated into other languages. Then we download and ship those with the app.

Reply:
Will do, thank!